css-inline/src/main.rs

```diff
@@ -47,6 +47,9 @@ OPTIONS:
 
     --extra-css
         Additional CSS to inline.
+
+    --styles-as-attributes
+        Will insert styles as attributes
```

Comment:
```suggestion
        Will insert styles as separate attributes.
```

For consistency with other lines above. Also, I think adding "separate" makes it a bit easier to understand what would the end result look like